src/braket/experimental/algorithms/chsh_inequality/chsh_inequality.py

```diff
@@ -27,9 +27,9 @@
 def create_chsh_inequality_circuits(
     qubit0: Qubit = 0,
     qubit1: Qubit = 1,
+    a1: float = np.pi / 2,
```

Comment:
This is what stopped me from making the same change. I don't think we can just change the parameter order since the change is backward incompatible. We need to talk to whoever owns this to figure it out.

Reply:
The other option here would be to inline ignore the ordering error from Flake.

Reply:
I don't think we can ignore the underlying problem. The parameters are (or seem to be) in the wrong order. This is experimental, so may be we just change them and not worry about backward compatibility, but we should check with whoever wrote/code reviewed this to make sure that changing it makes sense.

Reply:
Sure.

I had already check with the uses cases of where it appeared including:
https://github.com/aws-samples/amazon-braket-algorithm-library/blob/main/notebooks/textbook/CHSH_Inequality.ipynb
https://github.com/aws-samples/amazon-braket-algorithm-library/blob/main/src/braket/experimental/algorithms/chsh_inequality/chsh_inequality.py#L41
and the order differs in these instances.

Reply:
I wrote this code originally, and I would be in favor of preventing positional arguments. Its very confusing without having to name them explicitly. I'm okay with either a1, b1,a2,b2 or a1,a2,b1,b2. 
```python
def funct(a, b, *, a1, a2, b1, b2):
```
(I just learned this syntax prevent positional args!)